File: fizzbuzz.py
for i in range(100):
    # １から40まで
    i += 1
    s = ""
    if i % 3 == 0:
        s += "Fizz"
    if i % 5 == 0:
        s += "Buzz"
    if s == "":
        s += str(i)
    print(s)

File: fizzbuzz.js
for(var i = 1; i < 101; i++){
  var s = "";
  if(!(i % 3)){
    s += "Fizz";
  }
  if(!(i % 5)){
    s += "Buzz";
  }
  if(s == ""){
    s = i.toString()
  }
  console.log("%s", s);
}
